extensions/kind/src/create-cluster.ts

```diff
@@ -140,24 +141,30 @@ export async function createCluster(
 
   // grab http host port
   let httpHostPort = 9090;
-  if (params['kind.cluster.creation.http.port']) {
-    httpHostPort = params['kind.cluster.creation.http.port'];
+  if (params['kind.cluster.creation.http.port'] && typeof params['kind.cluster.creation.http.port'] === 'number') {
+    httpHostPort = Number(params['kind.cluster.creation.http.port']);
   }
 
   // grab https host port
   let httpsHostPort = 9443;
-  if (params['kind.cluster.creation.https.port']) {
-    httpsHostPort = params['kind.cluster.creation.https.port'];
+  if (params['kind.cluster.creation.https.port'] && typeof params['kind.cluster.creation.https.port'] === 'number') {
+    httpsHostPort = Number(params['kind.cluster.creation.https.port']);
   }
 
   let ingressController = false;
-
+  // The params['kind.cluster.creation.ingress'] can be only "on" or "undefined"
   if (params['kind.cluster.creation.ingress']) {
-    ingressController = params['kind.cluster.creation.ingress'];
+    ingressController = true;
```

Comment:
I think it's unspecified there so I would keep the reading of the parameter if provided to avoid any surprise if for example we receive it with false or off one day